src/test/java/com/datastax/oss/protocol/internal/PrimitiveSizesTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2017-2017 DataStax Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.protocol.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import org.junit.Test;
+
+public class PrimitiveSizesTest {
+
+  /**
+   * Check that {@link PrimitiveSizes#encodedUTF8Length} returns the correct value for every
+   * possible Unicode code point.
+   *
+   * <p>We compare with {@link String#getBytes(Charset)}, since that's how we encode strings in the
+   * driver.
+   */
+  @Test
+  public void should_measure_size_of_encoded_codepoint() {
```

Comment:
This test is great, but how long does it take to execute?

Reply:
It's pretty fast (around 2s from the IDE on my laptop). There are only a little over 1 million codepoints in Unicode.